vlab-research/fly
# Create initial web survey framework and render basic question types in survey

### What changes have you made?
- created a repository folder named `websurvey`
- set up an initial web app in svelte.js
- created a form that renders some hardcoded JSON data from the Typeform API
- imported the [svelte routing library](https://github.com/EmilTholin/svelte-routing)
- created components for two initial question types: `short_text` and `multiple_choice`
- implemented dynamic routing so that on index change the pathname updates along with the component
- added navigation functionality to the homepage and form buttons
- created a thankyou screen that displays at the end of the survey

### Which issue(s) does this PR fix?
[#49](https://github.com/vlab-research/fly/issues/49?notification_referrer_id=NT_kwDOAywRzbMyNzEwNDM5MDYxOjUzMjE5Nzg5#event-5646248627), [#50](https://github.com/vlab-research/fly/issues/50#event-5646245845), [#51](https://github.com/vlab-research/fly/issues/51#event-5646258083), [#58](https://github.com/vlab-research/vlab/issues/58#issuecomment-962013491), [#59](https://github.com/vlab-research/vlab/issues/59#issuecomment-966487162) 

### How to test
- Pull down the branch and run `npm install && npm run dev`
- When the user navigates to "/",  they are shown the homepage
- On submit of the form the user should be able to move to the next question in linear order
- On submit of the form the pathname should change to the corresponding `ref`
- When the user submits the last question of the survey, they are shown the thankyou screen
